views/admin.php.inc

```diff
@@ -1,183 +1,173 @@
+										<div class="description">
+                                            Select to show on all except the exceptions<br>
+											De-select to select the specific pages
+                                        </div>
+                                    </div>
+                                    <div class="form-item form-type-textarea div_hide_specific">
+                                        <label for="hide_oncustom">Except on pages:</label>
+                                        <div class="form-textarea-wrapper resizable textarea-processed resizable-textarea">
+                                            <textarea class="form-textarea hide_specific" name="hide_oncustom" id="hide_oncustom" cols="40" rows="6"><?php if (!empty($display_opts->hide_oncustom)) {
+                                                    $whitelist = json_decode($display_opts->hide_oncustom);                                                     
+                                                    foreach ($whitelist as $page) { echo $page."\r\n"; }
+                                            } else {
+                                            } ?></textarea>
+                                            <div class="grippie"></div>
+                                        </div>
+                                        <div class="description">
+                                            Add URLs to pages in which you would like to hide the widget.<br>
+                                            Put each URL in a new line.
+                                        </div>
+                                    </div>
+                                    <div class="form-item form-type-checkbox div_show_specific">
+                                        <?php
+                                            $checked = false;
+                                            if (!is_null($display_opts)) {
+                                                if (!$display_opts->show_onfrontpage) {
+                                                    $checked = true;
+                                                }
+                                            }
+                                        ?>
+                                        <input type="checkbox" id="show_onfrontpage" name="show_onfrontpage" value="1" class="checkbox show_specific" 
+                                            <?php echo ($checked)?'checked':'';?> />
+                                        <label class="option" for="show_onfrontpage">Show on front page</label>
+                                    </div>
+                                    <div class="form-item form-type-checkbox div_show_specific">
+                                        <?php
+                                            $checked = false;
+                                            if (!is_null($display_opts)) {
+                                                if (!$display_opts->show_oncategory) {
```

Comment:
This should be `if ($display_opts->show_oncategory) {`

Reply:
Or if we're changing to `taxonomy` (which makes much more sense since Backdrop uses `Taxonomy`), we can change `show_oncategory` to `show_ontaxonomy`.

Reply:
Done. Good suggestion